server/autotest_server/testers/haskell/settings_schema.json

```diff
@@ -1,12 +1,29 @@
 {
   "type": "object",
+  "required": [
+    "env_data"
+  ],
   "properties": {
     "tester_type": {
       "type": "string",
       "enum": [
         "haskell"
       ]
     },
+    "env_data": {
+      "title": "Haskell environment",
+      "type": "object",
+      "required": [
+        "resolver_version"
+      ],
+      "properties": {
+        "resolver_version": {
+          "title": "Resolver version",
+          "type": "string",
+          "enum": []
```

Comment:
This shouldn't be an enum, it should just be a string field. The point is to allow the user to specify different stack LTS versions.

Stack LTS versions aren't like Python versions. You don't need to pre-install them, they will get installed when `stack build` is run. However it _is_ good to have a default value for this string field which is equal to the default LTS version (lts-16.17).